change/@ni-jasmine-parameterized-544a8033-fa3b-4201-872c-b6f070a48513.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you update the README to document the usage: https://github.com/ni/nimble/blob/4d4c614052de37bf6f6f42f2d4c55e6ac87de3dd/packages/jasmine-parameterized/README.md#usage